Dockerfile

```diff
@@ -55,7 +55,7 @@ RUN set -xe \
                         | awk 'system("[ -e /usr/local/lib/" $1 " ]") == 0 { next } { print "so:" $1 }' \
         )" \
         && apk add --virtual .erlang-rundeps $runDeps lksctp-tools \
-        && cd / && git clone -b ${EMQX_VERSION} https://github.com/emqx/emqx-rel /emqx \
+        && cd / && git clone -b ${EMQX_DEPS_DEFAULT_VSN} https://github.com/emqx/emqx-rel /emqx \
```

Comment:
we should change to `EMQX_VERSION=v3.0-rc.5` as for emqx-rel branch name or git tag
and add `ENV EMQX_DEPS_DEFAULT_VSN=v3.0-rc.5`